libraries/ESP8266WiFi/examples/RangeExtender-NAPT/RangeExtender-NAPT.ino

```diff
@@ -53,9 +53,10 @@ void setup() {
   }
   Serial.printf("\nSTA: %s (dns: %s / %s)\n", WiFi.localIP().toString().c_str(), WiFi.dnsIP(0).toString().c_str(), WiFi.dnsIP(1).toString().c_str());
 
-  // give DNS servers to AP side
-  dhcpSoftAP.dhcps_set_dns(0, WiFi.dnsIP(0));
-  dhcpSoftAP.dhcps_set_dns(1, WiFi.dnsIP(1));
+  // by default, DNS option will point to the interface IP
+  // instead, point it to the real DNS server
+  // (notice that only a single IP is supported)
```

Comment:
[This is not totally true](https://www.rfc-editor.org/rfc/rfc2132.html#section-3.8) and thanks for this comment.
Multiple option 6 (DHCP_OPTION_DNS_SERVER) seems to be allowed by RFC and it was not supported by our current code. I'm not sure it is useful to change anything but this comment, here or there.

Reply:
...supported by the *implementation*. should there be `add(code, ip4_addr_t[])` and list of dns servers instead of just one?

Reply:
Nope, I guess, until someone demonstrates the need for this :]